### PR TITLE
librbd: object-map rebuild fail, add error information

### PR DESCRIPTION
**Pull request by x11562**
Signed-off-by: Na Xie <xie.na@h3c.com>

### CODE REVIEW

File: src/librbd/Operations.cc [hunk 1]
```diff
@@ -426,6 +426,7 @@ void Operations<I>::execute_rebuild_object_map(ProgressContext &prog_ctx,
     return;
   }
   if (!m_image_ctx.test_features(RBD_FEATURE_OBJECT_MAP)) {
+    lderr(cct) << "rbd object-map rebuild,image must support object-map feature" << dendl;
```
Comment (dillaman) [inline src/librbd/Operations.cc]: Minor: perhaps just shorten this to "image must support object-map feature"

**Comment by x11562** [comment]
@dillaman  fixed
